component_registry_bindings/iterators.py

```diff
@@ -95,7 +96,7 @@ def make_response_iterable(response, retrieve_list_fn, *args, **kwargs):
             kwargs.pop("limit", None)
             kwargs.pop("offset", None)
             param = getattr(response, param_name, None)
-            if param is None:
+            if param in (None, UNSET):
```

Comment:
guessing this is the temporary guard as prev discussed (until openapi definition is fixed in corgi) ?

Reply:
Exactly.